pyproject.toml

```diff
@@ -62,6 +62,7 @@ build = "^1.0.3"
 setuptools = "^69.1.1"
 httpx = ">=0.25.1"
 twine = ">=4.0.0"
+tomlkit = "^0.12.4"
```

Comment:
Is this a new dependency or already included by one of the packages? Is there a reason for tomlkit rather than the base toml library?

Reply:
tomllib was introduced in 3.11 and it intentionally only includes read/parse. There's no write.

Reply:
tomllib was introduced in 3.11 and it intentionally only includes read/parse. There's no write.